packages/abi-coder/src/index.ts

```diff
@@ -1,21 +1,17 @@
-export * from './coders/abstract-coder';
-export { default as Coder } from './coders/abstract-coder';
-export { default as ArrayCoder } from './coders/array';
-export { default as B256Coder } from './coders/b256';
-export { default as B512Coder } from './coders/b512';
-export { default as BooleanCoder } from './coders/boolean';
-export { default as ByteCoder } from './coders/byte';
-export { default as EnumCoder } from './coders/enum';
-export { default as NumberCoder } from './coders/number';
-export { default as StringCoder } from './coders/string';
-export { default as StructCoder } from './coders/struct';
-export { default as TupleCoder } from './coders/tuple';
-export { default as U64Coder } from './coders/u64';
-export { default as VecCoder } from './coders/vec';
-export * from './utilities';
-export * from './fragments/fragment';
-export { default as FunctionFragment } from './fragments/function-fragment';
-export { default as Interface } from './interface';
-export { default as AbiCoder } from './abi-coder';
```

Comment:
`AbiCoder` is no longer being exported from the package because it's now an implementation detail used inside of `Interface` to provide encoding/decoding of everything `JsonAbi`-related.
`Interface` is now the single entry point for anything `JsonAbi`-related.